firefox-ios/Client/Experiments/Experiments.swift

```diff
@@ -247,7 +249,12 @@ enum Experiments {
             .with(featureManifest: FxNimbus.shared)
             .with(commandLineArgs: CommandLine.arguments)
             .with(recordedContext: nimbusRecordedContext)
-            .build(appInfo: getAppSettings(isFirstRun: isFirstRun))
+            .build(
+                appInfo: getAppSettings(
+                    isFirstRun: isFirstRun,
+                    remoteSettingsService: remoteSettingsService
```

Comment:
this looks suspiciously too easy 🤔 i feel like im missing something...

Reply:
I will build your PR from AS against this later, but I think this is all we need to do. It does feel too simple though, I will give you that hahaha